picard/script.py

```diff
@@ -413,6 +413,13 @@ def func_unset(parser, name):
     """Unsets the variable ``name``."""
     if name.startswith("_"):
         name = "~" + name[1:]
+    # Allow wild-card unset for certain keys
+    if name in ('performer:*', 'comment:*', 'lyrics:*'):
```

Comment:
I chose the above ones just to include the common 'tag:name' type tags that we have.